api/app/controllers/api/v1/makers_controller.rb

```diff
@@ -3,8 +3,13 @@ module V1
     # makers controller
     class MakersController < ApplicationController
 
+      INCLUDES = %w(
+        users
+      ).freeze
+
       resourceful! Maker, authorize_options: { except: [:index, :show] } do
         Maker
+          .includes(:users)
```

Comment:
This seems wrong to me, per my comment on the pivotal tracker story. We don't want to expose users on the serialized maker, as there's no reason for the client application to know which users belong to a given maker (it's the other side of the relationship that's more useful)